build.gradle

```diff
@@ -61,16 +69,46 @@ repositories {
     mavenCentral()
 }
 
+// Write the plugin's classpath to a file to share with the tests
+task createClasspathManifest {
+  def outputDir = file("$buildDir/$name")
+
+  inputs.files sourceSets.main.runtimeClasspath
+  outputs.dir outputDir
+
+  doLast {
+    outputDir.mkdirs()
+    file("$outputDir/plugin-classpath.txt").text = sourceSets.main.runtimeClasspath.join("\n")
+  }
+}
+
+tasks.test.dependsOn tasks.createClasspathManifest
+if (findProject(':vendor') != null) {
+  def vendorModelRegistry = project.evaluationDependsOn(':vendor').getModelRegistry()
+
+  // We just want this path to be realize so we ignore the return type
+  vendorModelRegistry.realizeNode(ModelPath.path('components'))
+  def protoc = vendorModelRegistry.find(ModelPath.path('components.protoc.binaries.protocExecutable'), ModelType.of(NativeExecutableBinarySpec.class))
```

Comment:
Please limit your line length to 100, and follow [this guide](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit) for line breaks.